### PR TITLE
Ensure docs build

### PR DESCRIPTION
**Pull request by CAD97**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -33,4 +33,26 @@ jobs:
...
+      uses: actions/checkout@v2
+
+    - name: Install Rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly
+        profile: minimal
+
+    - name: Rustdoc
+      run: cargo doc --all-features
```
Comment (matklad) [inline .github/workflows/ci.yaml]: Let's put that as a job into an existing step. The crate is tiny, so minimizing the size of .yaml seems more important than increasing ||

**Comment by matklad** [comment]
bors d+

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (CAD97) [reply .github/workflows/ci.yaml]: The reason I've put this into a separate job is that it requires rust nightly for intra doc links. Unless that has reached stable and I didn't notice?

**Comment by CAD97** [comment]
Hmm, this should have failed the build, but didn't.

```
D:\repos\rust-analyzer\text-size>git checkout master
Already on 'master'
Your branch is up to date with 'upstream/master'.

D:\repos\rust-analyzer\text-size>cargo +nightly doc
warning: `[TextSized]` cannot be resolved, ignoring it.
  --> src\size.rs:47:62
   |
47 |     /// want to be usable in this constructor can implement [`TextSized`].
   |                                                              ^^^^^^^^^^^ cannot be resolved, ignoring
   |
   = note: `#[warn(intra_doc_link_resolution_failure)]` on by default
   = help: to escape `[` and `]` characters, just add '\' before them like `\[` or `\]`

    Finished dev [unoptimized + debuginfo] target(s) in 0.02s
```

**Comment by CAD97** [comment]
That is fixed by #32.

**Pull request merged**